### ISSUE TITLE
[Bug] Shows old and read notifications

### ISSUE THREAD

**Issue by mathmakgakpak** [YTVanced/Vanced#1138]
**Bug description**
Shows old and read notifications.

**Variant**
Non-Root

**Vanced version**
16.20.35

**Android version**
11

**Device**
Huawei P20

**Steps to Reproduce**
Wait

**Expected behavior**
Show notification only once

_Attach images or videos or logcats in case of crashes if possible. These can be helpful in solving the issue._
https://user-images.githubusercontent.com/44949126/128146453-1d3c1f24-b966-4b5e-a6dd-c89644ff6f2b.mp4

**Comment by milindgoel15** [comment]
Duplicate of #1107

**Comment by mathmakgakpak** [comment]
i cant stop vanced from doing that so i need to uninstall it

**Comment by mathmakgakpak** [comment]
sorry for duplicating issues i searched for open issues only cuz it still happened for me so i thought that it should be opened

**Comment by milindgoel15** [comment]
It's caused by microG latest updates and hence, it's not under our control because it's not made by us. microG belongs to microG team(marvin is the dev of it). Vanced microG just pulls and rebase commits from head repo.
